vl-convert-python/src/lib.rs

```diff
@@ -145,7 +145,7 @@ fn vegalite_to_svg(
         Ok(vega_spec) => vega_spec,
         Err(err) => {
             return Err(PyValueError::new_err(format!(
-                "Vega to SVG conversion failed:\n{}",
+                "Vega-Lite to SVG conversion failed:\n{}",
```

Comment:
A bunch of these error messages were copy pasted without being updated to the correct From/To values